src/documents/LocalizationInThePreferencesFramework.md

```diff
@@ -232,3 +232,29 @@ The strings can also be accessed directly, outside of the context of IoC referen
 ```javascript
 that.msgLookup.lookup(value); // where value is either the string name or the key in the stringArrayIndex to lookup.
```

Comment:
Looks like it still is. https://github.com/fluid-project/infusion/blob/master/src/framework/preferences/js/Panels.js#L18-L53